src/draw/grouping.typ

```diff
@@ -13,6 +13,28 @@
 
 #import "transformations.typ": move-to
 
+/// Calculates the intersections between multiple paths and create one anchor per intersection point.
```

Comment:
Typo, my suggestions are either
"Calculates [...] and creates [...]" or
"Calculate  [...] and create [...]"